Merge remote-tracking branch 'refs/remotes/origin/master' into share

# Conflicts:
#	django/django_test/mytestsite/home/__pycache__/urls.cpython-37.pyc
#	django/django_test/mytestsite/home/__pycache__/views.cpython-37.pyc
#	django/django_test/mytestsite/home/urls.py
#	django/django_test/mytestsite/mytestsite/__pycache__/settings.cpython-37.pyc
#	django/django_test/mytestsite/mytestsite/__pycache__/urls.cpython-37.pyc
#	django/django_test/mytestsite/mytestsite/__pycache__/wsgi.cpython-37.pyc
#	django/django_test/mytestsite/mytestsite/settings.py
#	django/django_test/mytestsite/mytestsite/urls.py
#
# It looks like you may be committing a merge.
# If this is not correct, please remove the file
#	.git/MERGE_HEAD
# and try again.


# Please enter the commit message for your changes. Lines starting
# with '#' will be ignored, and an empty message aborts the commit.
#
# On branch share
# Your branch is up-to-date with 'origin/share'.
#
# All conflicts fixed but you are still merging.
#
# Changes to be committed:
#	new file:   .gitignore
#	modified:   django/django_test/mytestsite/accounts/templates/registration/login.html
#
# Changes not staged for commit:
#	modified:   django/django_test/mytestsite/home/__pycache__/__init__.cpython-37.pyc
#	modified:   django/django_test/mytestsite/home/__pycache__/urls.cpython-36.pyc
#	modified:   django/django_test/mytestsite/home/__pycache__/views.cpython-36.pyc
#	modified:   django/django_test/mytestsite/home/templates/home/index.html
#	modified:   django/django_test/mytestsite/home/views.py
#	modified:   django/django_test/mytestsite/mytestsite/__pycache__/__init__.cpython-37.pyc
#	modified:   django/django_test/mytestsite/mytestsite/__pycache__/settings.cpython-36.pyc
#	modified:   django/django_test/mytestsite/mytestsite/__pycache__/urls.cpython-36.pyc
#	modified:   django/django_test/mytestsite/upload/__pycache__/__init__.cpython-37.pyc
#	modified:   django/django_test/mytestsite/upload/__pycache__/urls.cpython-36.pyc
#	modified:   django/django_test/mytestsite/upload/__pycache__/urls.cpython-37.pyc
#	modified:   django/django_test/mytestsite/upload/__pycache__/views.cpython-36.pyc
#	modified:   django/django_test/mytestsite/upload/__pycache__/views.cpython-37.pyc
#	modified:   django/django_test/mytestsite/upload/templates/upload/upload.html
#	modified:   django/django_test/mytestsite/upload/views.py
#	modified:   recolorAI/Recolor_3.0/recolorNet.py
#
# Untracked files:
#	django/django_test/mytestsite/home/templates/home/aboutus.htm
#	django/django_test/mytestsite/home/templates/home/contact.htm
#	django/django_test/mytestsite/home/templates/home/top_photos.htm
#	django/django_test/mytestsite/model.json
#	django/django_test/mytestsite/upload/files/6v14hm.jpg
#	django/django_test/mytestsite/upload/recolorMod/
#	django/django_test/mytestsite/upload/result/
#

